objects/scenery_small/shrub/nedesigns.scenery_small.shrub.fern/object.json

```diff
@@ -0,0 +1,29 @@
+{
+    "id": "nedesigns.scenery_small.shrub.fern",
+    "authors": ["Chris Sawyer", "Simon Foster"],
+    "version": "1.0",
+    "originalId": "09F55701|FERN3   |35CBCF47",
```

Comment:
Isn't this straight vanilla?

Reply:
It’s the miniaturized giant fern, I don’t think this is a vanilla object but also dose see an author for the custom version